torch/distributed/optim/named_optimizer.py

```diff
@@ -67,8 +66,8 @@ def __init__(
         optimizer_class: optim.Optimizer,
         param_groups: Optional[Collection[Mapping[str, Any]]] = None,
         module: Optional[nn.Module] = None,
-        *args,
-        **kwargs,
+        *args: tuple[Any, ...],
+        **kwargs: dict[str, Any],
```

Comment:
`kwargs` should be typed `Any` unless their values are `dict`. And `args` should be `Any`

Reply:
@cyyever they can't be expanded unless the keys are string so it should be fine

Reply:
@cyyever Actually, doing [str, Any] is safer and I'm pretty sure preferrable as it can detect those kind of type errors ahead of time

Reply:
By Python syntax, their keys are of course strings.

Reply:
Yeah so this should be the appropriate typing for kwargs.